src/utils/dates.ts

```diff
@@ -10,3 +10,13 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset: number) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+export function is24HoursLater(purchaseDate: Date): boolean {
+	const currentTime = new Date();
+
+	const timeElapsedInMilliseconds = Math.abs(
```

Comment:
I think this math is a bit incorrect. Using `Math.abs` here allows this to evaluate "24 hours ago" the same as "24 for hours from now"

Reply:
You are absolutely right! I always knew simple math would haunt me 🤦🏽‍♀️😂

Reply:
This is the kind of thing unit tests are very _very_ good at. Simple math haunts us all 🙂

Reply:
This is an interesting take! I had to wrap my head around this one because I initially thought if purchase date is (logically) always in the past, the 24 hour difference should technically always be positive and then Math.abs wouldn't make a difference?! Then I began to think, there could be cases perhaps if purchase date was for some reason, set to the future and the function is24hoursLater wouldn't be checking the correct logic...in any case, I think it is really valuable to capture the direction of the change in time either way!!